src/borg/crypto.pyx

```diff
@@ -445,24 +459,24 @@ cdef class _AEAD_BASE:
 
     def set_iv(self, iv):
         self.blocks = 0  # number of cipher blocks encrypted with this IV
-        for i in range(12):
+        for i in range(self.iv_len):
             self.iv[i] = iv[i]
 
     def next_iv(self):
         assert self.blocks < 2**32
         # we need 16 bytes for increment_iv:
-        last_iv = b'\0\0\0\0' + self.iv[:12]
+        last_iv = b'\0' * (16 - self.iv_len) + self.iv[:self.iv_len]
```

Comment:
How can we standardize on 16 bytes if some ciphers use 12 bytes? Any why would we want to try?